libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4091,6 +4091,15 @@ void GCS_MAVLINK::handle_common_message(const mavlink_message_t &msg)
         AP_CheckFirmware::handle_msg(chan, msg);
         break;
 #endif
+
+#if HAL_EFI_ENABLED
```

Comment:
```suggestion
#if HAL_EFI_MAV_ENABLED
```

This could go either way.

Reply:
Just to be sure: This should be `#if AP_EFI_MAV_ENABLED` right?

Reply:
> Just to be sure: This should be `#if AP_EFI_MAV_ENABLED` right?

Yep.  We're canonicalising on `AP_` but we're not sweeping for the older stuff.